independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java

```diff
@@ -987,8 +987,13 @@ Builder disposer(DisposerInfo disposer) {
             return this;
         }
 
+        /**
+         * @deprecated use {@link #alternative(boolean)} and {@link #priority(Integer)};
+         *             this method will be removed at some time after Quarkus 3.6
```

Comment:
Quarkus 3.6??

Reply:
I'm thinking roughly 6 months after the Quarkus release in which this would happen, which is 3.0. Given our roughly monthly release cadence, that's 3.6. Do you have another preference?